src/test/java/org/fundacionjala/coding/jorge/FizzBuzzTest.java

```diff
@@ -0,0 +1,49 @@
+package org.fundacionjala.coding.jorge;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+
+/**
+ *
+ */
+public class FizzBuzzTest {
+
+    private FizzBuzz fiBu;
+    private Object[] expected;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "expected" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)